conftest.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from users.models import Role, User, Team
+
+
+@pytest.fixture
+def new_team_factory(db):
```

Comment:
I think you can do it like:
```suggestion
def team_1(db):
```
and get rid of the team_1 fixture in lines # 15-17

Reply:
why?? I dont agree, you are losing the whole point with creating factory